app/controllers/api/v2/compute_resources_controller.rb

```diff
@@ -15,7 +15,7 @@ class ComputeResourcesController < V2::BaseController
       before_filter :find_resource, :only => [:show, :update, :destroy, :available_images, :associate,
                                               :available_clusters, :available_flavors, :available_folders,
                                               :available_networks, :available_resource_pools, :available_security_groups, :available_storage_domains,
-                                              :available_zones]
+                                              :available_zones, :available_storage_pods]
```

Comment:
I do not have any ideas how that could be refactored. Should I raise a Redmine for this?